R/data_unanchored.R

```diff
@@ -0,0 +1,20 @@
+#' Dummy Single-arm Trial Data for Unanchored MAIC
+#'
+#' These datasets contains adsl_sat, adtte_sat, adrs_sat,
+#' agd_sat (AgD of effect modifiers), pseudo_ipd_sat (AgD, tte data)
+#'
+#' @format A data frame with xx rows and x columns:
+#'   \describe{
+#'     \item{id}{The ID of the patients.}
+#'   }
+#'
+#' @source These datasets are generated by the dummy_unanchored.R script.
+#'
+#' @keywords dataset
+#'
+"adsl_sat"
+"adtte_sat"
+"pseudo_ipd_sat"
+"agd_sat"
+"centered_ipd_sat"
+"adrs_sat"
```

Comment:
Normally we'd need to document each data set, about all the columns and what they contain.